utils/python/CIME/test_status.py

```diff
@@ -79,6 +79,9 @@ def __init__(self, test_dir=None, test_name=None, no_io=False):
         Create a TestStatus object
 
         If test_dir is not specified, it is set to the current working directory
+
+        no_io is intended only for testing, and should be kept False in
```

Comment:
This is what I inferred from its current usage... feel free to correct me if I'm wrong